Tests/test_content.py

```diff
@@ -913,58 +833,48 @@ def execute_testing(tests_settings, server_ip, mockable_tests_names, unmockable_
+                                  demisto_pass, demisto_api_key)
+                logging_manager.execute_logs()
             proxy.configure_proxy_in_demisto(username=demisto_user, password=demisto_pass, server=server)
 
             # reset containers after clearing the proxy server configuration
-            reset_containers(server, demisto_user, demisto_pass, prints_manager, thread_index)
+            reset_containers(server, demisto_user, demisto_pass)
 
-        prints_manager.add_print_job("\nRunning mock-disabled tests", print, thread_index)
+        logging_manager.info("Running mock-disabled tests\n", real_time=True)
         executed_in_current_round, unmockable_tests_queue = initialize_queue_and_executed_tests_set(unmockable_tests)
         while not unmockable_tests_queue.empty():
             t = unmockable_tests_queue.get()
             executed_in_current_round = update_round_set_and_sleep_if_round_completed(executed_in_current_round,
-                                                                                      prints_manager,
-                                                                                      t,
-                                                                                      thread_index,
-                                                                                      unmockable_tests_queue)
+                                                                                      t)
             run_test_scenario(unmockable_tests_queue, tests_settings, t, proxy, default_test_timeout,
                               skipped_tests_conf, nightly_integrations, skipped_integrations_conf, skipped_integration,
                               is_nightly, filtered_tests, skipped_tests, secret_params,
                               failed_playbooks, playbook_skipped_integration, unmockable_integrations,
                               succeed_playbooks, slack, circle_ci, build_number, server, build_name,
-                              server_numeric_version, demisto_user, demisto_pass, demisto_api_key,
-                              prints_manager, thread_index, is_ami)
-            prints_manager.execute_thread_prints(thread_index)
+                              server_numeric_version, demisto_user, demisto_pass, demisto_api_key, is_ami)
+            logging_manager.execute_logs()
 
-    except Exception as exc:
-        if exc.__class__ == ApiException:
-            error_message = exc.body
-        else:
-            error_message = f'~~ Thread {thread_index + 1} failed ~~\n{str(exc)}\n{traceback.format_exc()}'
-        prints_manager.add_print_job(error_message, print_error, thread_index)
-        prints_manager.execute_thread_prints(thread_index)
-        failed_playbooks.append(f'~~ Thread {thread_index + 1} failed ~~')
+    except Exception:
+        logging_manager.exception('~~ Thread failed ~~')
+        logging_manager.execute_logs()
```

Comment:
Do we need this `execute_logs()` here and in the `try` clause if you are invoking it in the `finally` clause on line 870?